awslabs/aws-lambda-powertools-typescript
# docs(tracer): extract code examples

## Description of your changes

Extract code examples in the tracer documentation

Changes:
- group extracted examples under `docs/examples/core/tracer/` folder
- fix indents to match the projects code style of 2 spaces
- fix the sam `template.yml` example, by adding missing fields
- fix missing import for `Context` tracing https requests example

Resolves #729

No specific dependencies, but improvements could be made to validate the code examples and allow for this to be a scalable option.

### How to verify this change

Run `npm run docs-runLocalDocker` and the code examples in the docs should render the same.

### Related issues, RFCs

- #729

### PR status

***Is this ready for review?:*** YES  
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist


---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
